### DISCUSSION TITLE
Problem to compile program

### DISCUSSION THREAD

**Discussion by Fracob** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#359] [General]
Did anyone had issues like this compiling code with VSC 

" Compiling .pio/build/nodemcuv2/libff6/WebSockets/libb64/cencode.c.o
.pio/libdeps/nodemcuv2/WebSockets/src/SocketIOclient.cpp: In member function 'void SocketIOclient::loop()':
.pio/libdeps/nodemcuv2/WebSockets/src/SocketIOclient.cpp:175:22: warning: 'void WebSocketsClient::loop()' is deprecated [-Wdeprecated-declarations]
  175 |    WebSocketsClient::loop();
      |                      ^~~~
In file included from .pio/libdeps/nodemcuv2/WebSockets/src/SocketIOclient.cpp:9:
.pio/libdeps/nodemcuv2/WebSockets/src/WebSocketsClient.h:75:10: note: declared here
   75 |     void loop(void) __attribute__((deprecated)) {}
      |          ^~~~
.pio/libdeps/nodemcuv2/WebSockets/src/SocketIOclient.cpp:175:27: warning: 'void WebSocketsClient::loop()' is deprecated [-Wdeprecated-declarations]
  175 |    WebSocketsClient::loop();
      |                           ^
In file included from .pio/libdeps/nodemcuv2/WebSockets/src/SocketIOclient.cpp:9:
.pio/libdeps/nodemcuv2/WebSockets/src/WebSocketsClient.h:75:10: note: declared here
   75 |     void loop(void) __attribute__((deprecated)) {}
      |          ^~~~ "

**Comment by visualapproach** [comment]
It's just warnings. We don't use that function. It's just a reminder.

**Reply by Fracob** [reply]
Ok. Thanks

**Reply by Fracob** [reply]
I have a limited edition lay-z-spa wirpool (like Miami 2021  but slightly larger) with square 2021 control and S100101 pump. I compiled and loaded the software on the nodemcu v3 board and performed the procedure to connect the PCBV2 board to the pump and to the wireless network. At start-up, music is heard and when it connects to the network, the pump display shows the IP address followed by the firmware data, then the display switches off and the degrees Centigrade, degrees Farenait and pump LEDs remain on but the keyboard does not respond to any commands. Even via the web page, no pump or bubble control action is possible (the control slide automatically switches from green to gray), no temperature indication. I tried both with and without  3 serie resistor of 560 ohm  between LLC data, clk and cs out and connector (DSP connector side) and with both LLC and TXS0108E converter. No control. I also checked the supply voltage and the possible presence of false contacts but all negative.
Suggestions on how to proceed are welcome.
Thank yo
![IMG_2233](https://user-images.githubusercontent.com/38158506/184556033-9b14aa34-31d5-481f-92ae-0bf7bb68c7ac.jpg)
![IMG_2234](https://user-images.githubusercontent.com/38158506/184556035-424e7ab0-b00a-4830-bd83-fb2775b71cd3.jpg)
![IMG_2238](https://user-images.githubusercontent.com/38158506/184556037-b499fb42-daca-471d-895a-243bc7dd1ba9.jpg)
![IMG_2241](https://user-images.githubusercontent.com/38158506/184556038-4b16c475-8201-41d1-8d45-69025e16b19c.jpg)
![IMG_2250](https://user-images.githubusercontent.com/38158506/184556039-eb440edd-9607-4c20-8f20-e7bc0b7804be.jpg)
![IMG_2275](https://user-images.githubusercontent.com/38158506/184556041-b4b4f0dd-fce5-450f-9d05-97a30f76139a.jpg)
![IMG_2276](https://user-images.githubusercontent.com/38158506/184556042-ce7169f6-6264-47df-943e-8ea6afc6d5f8.jpg)
![CF9141A6-F989-4AE8-9D53-6FDF081A2880](https://user-images.githubusercontent.com/38158506/184556060-14034834-87c1-42f9-8bb8-d727c3e65e6e.JPG)
![IMG_2252](https://user-images.githubusercontent.com/38158506/184556068-d484279a-f977-4519-9033-5f30e819fc3d.PNG)
![IMG_2251](https://user-images.githubusercontent.com/38158506/184556075-12be880f-ccfe-4dcc-884c-a435e76b7092.PNG)
u

**Reply by visualapproach** [reply]
Hi! I think you need to use the PCB_V2B (only software change) and change the wires accordingly. See development branch build instructions and if you want to read the details check #330
No resistors should be needed.

**Reply by Fracob** [reply]
With the PCB_V2B the software run and can control the pump but the display is flashing, I think that need put the 3 resistor in serie to the clk, data and cs of dsp cable.
Thank for the support.

**Reply by visualapproach** [reply]
Okay, try the resistors and if necessary the other type of LLC. You made your life easy by using headers! Good work!

**Reply by Fracob** [reply]
Problem solved changing the LLC (no resistor) everything run fine. Very good projects and very good support.
Thanks
